### ISSUE TITLE
0.44 winget release broke

### ISSUE THREAD

**Issue by jntrnr** [nushell/nushell#4368]
### Describe the bug

https://github.com/nushell/nushell/runs/5108310440?check_suite_focus=true

```
Run iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
Telemetry Settings
------------------
The Windows Package Manager Manifest Creator collects usage data in order to improve your experience.
The data is anonymous and collected only by Microsoft.
By default, telemetry is enabled but can be disabled by running `wingetcreate settings` and editing your settings file.

Retrieving latest manifest for Nushell.Nushell
Downloading and parsing: https://github.com/nushell/nushell/releases/download/0.44.0/nu_0_44_0_windows.msi...
Each new installer URL must have a single match to an existing installer based on installer type and architecture. The following installers failed to match an existing installer:
Multiple matches found for X64 Msi installer detected from the url: https://github.com/nushell/nushell/releases/download/0.44.0/nu_0_44_0_windows.msi
Error: Process completed with exit code 1.
```

### How to reproduce

1. See above

### Expected behavior

This is an automated release, so it should work as before

**Comment by jntrnr** [comment]
cc @TechWatching who did the original one

**Comment by jntrnr** [comment]
also tagging @OfficialEsco, who's been handling the winget package in the winget repo

any ideas on what might be causing the break?

**Comment by OfficialEsco** [comment]
Oh, wingetcreate probably didn't like us fixing the InstallerType from msi to wix
There will be a wingetcreate release soon™️(MS timeframe) that supports wix

I swear there was supposed to be a override option in wingetcreate but i cannot find it... Do you want me to update nushell for the time being?
cc @denelon

**Comment by jntrnr** [comment]
@OfficialEsco - ahh thanks for the update

Yes, if you don't mind, it'd be great to update nushell for the time being

**Comment by TechWatching** [comment]
> Oh, wingetcreate probably didn't like us fixing the InstallerType from msi to wix There will be a wingetcreate release soon™️(MS timeframe) that supports wix
> 
> I swear there was supposed to be a override option in wingetcreate but i cannot find it... Do you want me to update nushell for the time being? cc @denelon

Are you talking about the InstallerType in the manifest file or something else? Is it about [this issue](https://github.com/microsoft/winget-create/issues/222) in the wingetcreate package? I am not very familiar with installers packages for Windows so I am not sure to understand the problem.  If next release of WingetCreate will solve it, it's great. I am just asking for my information, to better understand what exactly was causing the problem.

**Comment by OfficialEsco** [comment]
We got notified of a issue where some WiX Installer's didn't accept the same InstallLocation switch as MSI, so we went looking for WiX Installers in the repo and changed the Installer Type from MSI to WIX in the Manifest so winget-cli can easily change the behavior of that particular Installer Type.

However by doing that we realized winget-create cannot differentiate WiX from MSI so we created the Issue and the Fix, aaand now the ball is back in Microsoft's hands, very few packages should be affected by this as 99.XXX% of the packages are submitted via our community YamlCreate.ps1 script

TLDR: This could have and can be prevented in the future if WinGet-Create was up to speed with WinGet-CLI

**Comment by OfficialEsco** [comment]
https://github.com/microsoft/winget-create/releases/tag/v0.5.0.1-preview should resolve the issue

**Comment by hustcer** [comment]
Maybe we can close this issue, too
